ui/js/models/repository.js

```diff
@@ -32,7 +32,11 @@ treeherder.factory('ThRepositoryModel', [
 
         const getOrderedRepoGroups = function () {
             if (!Object.keys(orderedRepoGroups).length) {
-                const groups = _.groupBy($rootScope.repos, function (r) { return r.repository_group.name; });
+                // group repos by 'repository_group.name'
```

Comment:
We could save a line with spread:

    const groups = $rootScope.repos.reduce((acc, repo, i, a, group = v => v.repository_group.name) => (
      { ...acc, [group(repo)]: [...acc[group(repo)] || [], repo] }
    ), {});

And slightly less terse names will make it easier to interpret, I think.  Would you mind doing them all like this?

Reply:
Thanks! I'll do that.